guide/how-it-works/sign-in-with-bitcoin.md

```diff
@@ -0,0 +1,161 @@
+
+The cryptography involved in this can also be used to generate public keys for other purposes, like a unique identifier that websites and applications can use for their account systems. This is similar to approaches like [Passkeys](https://developer.apple.com/passkeys/), which also look at replacing email & password combos with unique identifiers. [Decentralized identities](https://en.wikipedia.org/wiki/Decentralized_identifier) (DID) are another emerging approach.
+
+In the context of bitcoin, this allows users to sign in to services with their wallets and also use this connection to make low-friction payments.
+
+{% include tip/open.html color="blue" icon="info" label="All eyes on lightning" %}
+
+On this page, we only focus on lightning wallets, as they lend themselves much better to this type of user experience.
+
+{% include tip/close.html %}
+
+#### Support
+{:.no_toc}
+
+“Sign in with bitcoin” is not widely supported. It uses new technologies like [WebLN](https://webln.dev/#/) and [LNURL-auth](https://github.com/fiatjaf/lnurl-rfc/blob/luds/04.md) that are still maturing and looking for broader adoption. To try this out, you can use the [Alby](https://getalby.com/) browser extension wallet with a service like [Stacker News](https://stacker.news/) that allows you to spend satoshi to upvote news stories (more lightning apps can be found in the [bolt.fun maker directory](https://makers.bolt.fun/products)).
+
+### Wallet-service communication
+
+Your service and the user wallet need to be able to communicate with each other to exchange information. There are two basic ways to handle this—direct and bridged.
+
+#### Direct communication
+
+The smoothest user experience is when the wallet and the service can directly communicate with each other. For example, a browser wallet and a website can send messages back and forth (see [WebLN](https://www.webln.guide/introduction/readme)). This allows for convenient features like allowances for automatic approval of payments for services the user trusts.
+
+{% include picture.html
+   image = "/assets/images/guide/how-it-works/sign-in-with-bitcoin/direct-connection.png"
+   retina = "/assets/images/guide/how-it-works/sign-in-with-bitcoin/direct-connection@2x.png"
+   alt-text = "Browser window with a wallet extension installed"
+   width = 800
+   height = 411
+%}
+
+#### Bridged communication
+
+There is more friction if the user needs to be involved in this communication. However, this extra friction provides more user control and better security. In the visualization below, the service and user wallet cannot directly communicate with each other. Instead, the web application presents instructions as QR codes that the user needs to scan with their (mobile) wallet. The wallet then sends instructions to the application server, which updates the web application.
```

Comment:
An example app would be handy

Reply:
The support section above has tips for how to try this stuff out.